lib/internal/fs/utils.js

```diff
@@ -584,12 +584,15 @@ const validateBufferArray = hideStackFrames((buffers, propName = 'buffers') => {
   if (!ArrayIsArray(buffers))
     throw new ERR_INVALID_ARG_TYPE(propName, 'ArrayBufferView[]', buffers);
 
+  const copy = [];
```

Comment:
I think `const copy = buffers.slice();` will be faster than continually appending to an empty array.

Reply:
Ok, is it a shared opinion ? If that is the case, i will make the change :)